modules/core/src/main/java/org/apache/fluo/core/observer/ObserverUtil.java

```diff
@@ -99,8 +97,8 @@ public String getObserverId(Column col) {
         }
 
         @Override
-        public Set<Column> getObservedColumns(NotificationType nt) {
-          return Collections.emptySet();
```

Comment:
I think `Collections.emptySet()` is immutable, so this change is not necessary.  However making the change does not hurt anything.